src/QueryBuilder/QueryBuilder/helpers/selectOptions.js

```diff
@@ -36,6 +36,14 @@ const ArrayOperators = () => [
   { label: OPERATORS_LABELS.EMPTY, value: OPERATORS.EMPTY },
 ];
 
+const JsonbArrayOperators = () => [
+  { label: OPERATORS_LABELS.CONTAINS_ALL, value: OPERATORS.CONTAINS_ALL },
+  { label: OPERATORS_LABELS.NOT_CONTAINS_ALL, value: OPERATORS.NOT_CONTAINS_ALL },
+  { label: OPERATORS_LABELS.CONTAINS_ANY, value: OPERATORS.CONTAINS_ANY },
+  { label: OPERATORS_LABELS.NOT_CONTAINS_ANY, value: OPERATORS.NOT_CONTAINS_ANY },
+  { label: OPERATORS_LABELS.EMPTY, value: OPERATORS.EMPTY },
+];
```

Comment:
nit: assuming we'll never use different operators for jsonb arrays/ordinary arrays, we could remove this and just use `ArrayOperators()` on line 101

Reply:
@bvsharp  done